rules/autoconsent/ciaopeople-it.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ciaopeople.it",
+  "runContext": {
+    "urlPattern": "^https://(www\\.)?(ohga|lexplain|wamily|fanpage)\\.it/"
+  },
```

Comment:
I see a few other sites in the [source code](https://static-cmp22.ciaopeople.it/assets/r/2.0.44/sdk-gdpr.min.js), but I think it can change in the future. So I'd suggest to go without urlPattern and rely fully on `detectCmp` to support any future additions